src/templates/preview.html

```diff
@@ -1,7 +1,12 @@
 <div id="animated-preview-container" class="preview-container">
   <div class="canvas-container-wrapper minimap-container">
     <div class="preview-contextual-actions">
-      <div class="preview-contextual-action  original-size-button" rel="tooltip" data-placement="bottom">1x</div>
+      <div class="preview-drop-down">
+        <div class="preview-disable-overlay" rel="tooltip" data-placement="left"></div>
+        <div class="preview-contextual-action  original-size-button" rel="tooltip" data-placement="left">1x</div>
+        <div class="preview-contextual-action  best-size-button" rel="tooltip" data-placement="left"></div>
+        <div class="preview-contextual-action  full-size-button" rel="tooltip" data-placement="left">Full</div>
```

Comment:
Are you pro or cons dynamic adding in the DOM ? Thoses 3 can be appened at PreviewController's init.